test/helmfile/Dockerfile

```diff
@@ -0,0 +1,75 @@
+ARG IMAGE=containerbase/buildpack
```

Comment:
just add a simple test here, as you've bats tests, which cover the relevant parts.

https://github.com/containerbase/base/blob/88af9742be55f24b1c8fd6a6cc14f6bbdf7fe7f8/test/latest/Dockerfile#L163-L189

Reply:
remove this file